feature/system/ntp/tests/system_ntp_test/system_ntp_test.go

```diff
@@ -133,7 +136,16 @@ func TestNtpServerConfigurability(t *testing.T) {
 				ntp.SetEnabled(true)
 				for _, address := range testCase.addresses {
 					server := ntp.GetOrCreateServer(address)
-					server.SetSourceAddress(dutlo0Attrs.IPv4)
+					switch dut.Vendor() {
+					case ondatra.ARISTA:
```

Comment:
If this is a limitation for only one vendor, and if we require IPv6 device to IPv4 NTP server to be supported, then this is a deviation.

If we don't require IPv6 device to IPv4 NTP server and will instead always use the same address family, then we could make this the standard and don't need the switch dut.Vendor() statement.